ringvax/summary.py

```diff
@@ -38,7 +38,9 @@ def prepare_for_df(infection: dict) -> dict:
     return dfable
 
 
-def get_all_person_properties(sims: Sequence[Simulation]) -> pl.DataFrame:
+def get_all_person_properties(
```

Comment:
I got a little confused about these polars table manipulation functions. I think there are ways to do this more polaronically; I'll have a look in another PR

Reply:
Depending on what you mean, I either endorse this or urge caution.

I initially made each person a dataframe, then concatenated all those. But the overhead was extravagant

Reply:
See #45